integration-tests/src/main/kotlin/org/stellar/anchor/platform/ReferenceServerClient.kt

```diff
@@ -0,0 +1,24 @@
+package org.stellar.anchor.platform
+
+import okhttp3.HttpUrl
+import okhttp3.HttpUrl.Companion.toHttpUrlOrNull
+import okhttp3.Request
+
+class ReferenceServerClient(private val endpoint: String) : SepClient() {
+  fun health(checks: List<String>): java.util.HashMap<*, *> {
```

Comment:
💡 You could default the health check test to `"all"`:

```suggestion
  fun health(checks: List<String> = listOf("all")): java.util.HashMap<*, *> {
```

Reply:
Nice. Changed.